src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,54 @@
+package duke.task;
+
+import duke.dukeexception.DukeException;
+
+/**
+ * Represents the task of event format.
+ */
+public class Event extends Task {
+    private String at;
```

Comment:
Could this naming variable be more descriptive? 
```suggestion
    private String dateTime;
```